lib/rails/generators/channel/templates/index.coffee.erb

```diff
@@ -0,0 +1,9 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+# Uncomment out this code for a basic implementation of actioncable
+# @App ?= {}
+
+# Define the location of your websockets here
+# App.cable = Cable.createConsumer 'ws://localhost:28080'

```

Comment:
Perhaps use the application name as the toplevel object instead of `App`. So we'd have `@MyApp` here and could uncomment.